pyuvdata/uvbeam/tests/test_beamfits.py

```diff
@@ -6,9 +6,9 @@
 
 """
 from __future__ import absolute_import, division, print_function
```

Comment:
I don't think we need this line anymore, now that we're py3-only.

Reply:
yeah, these need to come out throughout the repo.

Reply:
I'll change these and make an issue to get the others out.